Mathlib/NumberTheory/NumberField/Norm.lean

```diff
@@ -26,13 +26,17 @@ open scoped NumberField BigOperators
 
 open Finset NumberField Algebra FiniteDimensional
 
-section rat
+section Rat
 
-theorem Algebra.coe_norm_int {K : Type*} [Field K] [NumberField K] (x : 𝓞 K) :
-    Algebra.norm ℤ x = Algebra.norm ℚ (x : K) :=
+variable {K : Type*} [Field K] [NumberField K] (x : 𝓞 K)
+
+theorem Algebra.coe_norm_int : Algebra.norm ℤ x = Algebra.norm ℚ (x : K) :=
   (Algebra.norm_localization (R := ℤ) (Rₘ := ℚ) (S := 𝓞 K) (Sₘ := K) (nonZeroDivisors ℤ) x).symm
 
-end rat
+theorem Algebra.coe_trace_int : Algebra.trace ℤ _ x = Algebra.trace ℚ K x :=
```

Comment:
Can you write the coercion explicitly with `↑` just to make the naming clear here?